File: Eventify.Core/Attendees/AttendeeSaveModel.cs
using System;
using Eventify.Common.Classes.Attendees;

namespace Eventify.Core.Attendees
{
	public class AttendeeSaveModel
    {
        public Guid? Id { get; set; }

        public Guid EventId { get; set; }

        public string Name { get; set; }

        public string LastName { get; set; }

        public string Code { get; set; }

        public string PaymentMethod { get; set; }

        public int? Participants { get; set; }

        public string? Notes { get; set; }

        public AttendeeType? AttendeeType { get; set; }
    }
}

File: Eventify.Web/ClientApp/src/app/models/attendees/attendee-save.model.ts
import { AttendeeType } from "./attendee.model";

export class AttendeeSave {
    id?: string;
    name?: string;
    lastName?: string;
    code?: string;
    paymentMethod?: string;
    participants?: number;
    notes?: string;
    eventId: string;
    attendeeType: AttendeeType;
}
